broker/emit.go

```diff
@@ -72,24 +73,24 @@ func EmitFx(w http.ResponseWriter, r *http.Request) (string, interface{}, error)
 	body := lib.ErrorByte(io.ReadAll(r.Body))
 	defer r.Body.Close()
 
-	log.Printf("[EmitFx] Request: %s", string(body))
+	log.Printf("Request: %s", string(body))
 	err = json.Unmarshal([]byte(body), &request)
 	if err != nil {
-		log.Printf("[EmitFx] error unmarshaling policy: %s", err.Error())
+		log.Printf("error unmarshaling policy: %s", err.Error())
 		return "", nil, err
 	}
 
 	uid := request.Uid
-	log.Printf("[EmitFx] Uid: %s", uid)
+	log.Printf("Uid: %s", uid)
 
 	paymentSplit = request.PaymentSplit
-	log.Printf("[EmitFx] paymentSplit: %s", paymentSplit)
+	log.Printf("paymentSplit: %s", paymentSplit)
```

Comment:
our constants are lowercase in the backend. Should we use a ToLower in this case? It should apply to other fields such as: provider (manual/fabrick), paymentSplit (monthly/yearly)